IOS/SwiftSeedProject/SwiftSeedProject/AppDelegate.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  AppDelegate.swift
+//  SwiftSeedProject
+//
+//  Created by Mugu on 3/17/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
```

Comment:
This is one of those funny things from iOS SDK. You actually need that UIWindow variable. I guess that it's not necessary if you do a library or console application, though.